src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -44,12 +46,17 @@
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
 
+import lombok.Setter;
+
 public abstract class IntegrationTestBase extends AbstractTestExecutionListener {
+    private static final int SECONDS_1 = 1000;
+    private static final int SECONDS_30 = 30000;
```

Comment:
While it is more readable.... it require change of variable name each time its value is changed.

Reply:
Yes, or to add new variable with new value :)